python_modules/dagster/dagster/_core/definitions/declarative_scheduling/scheduling_condition.py

```diff
@@ -185,6 +186,13 @@ def parent_newer() -> "ParentNewerCondition":
 
         return ParentNewerCondition()
 
+    @staticmethod
+    def newly_updated() -> "NewlyUpdatedCondition":
+        """Returns a SchedulingCondition that is true for an asset partition if it has been updated since the previous tick."""
+        from .operands import NewlyUpdatedCondition
+
+        return NewlyUpdatedCondition()
+
```

Comment:
Hmm given the description here I am now confused how this relates to the more general "SinceCondition" down the stack...